seqr/views/apis/family_api.py

```diff
@@ -48,6 +49,9 @@ def family_page_data(request, family_guid):
     if sample_models.filter(sample_type=Sample.SAMPLE_TYPE_RNA).exclude(rnaseqtpm=None):
         response['familiesByGuid'][family_guid]['hasRnaTpmData'] = True
 
+    if PhenotypePrioritization.objects.filter(individual__guid=individual_guid):
```

Comment:
this is inefficient, you are running one query per individual when you should be running one query total for the family

Reply:
The data is fetched when the users click the `Show Phenotype Prioritized Genes` button. Do the users usually click and see the high-ranked genes for multiple members in the family?

Reply:
this is an API endpoint returning all the data for a family. You need to return all the data a user would potentially see in that family. That means you need to return the data for all individuals in that family. How elese will you know which individuals should even have the button?

Reply:
Okay. It seems that my implementation of `hasPhenotypeGeneScores` isn't correct.

Reply:
Do you think we should have per individual `hasPhenotypeGeneScores`?  So that the individuals who don't have data won't show the button.

Reply:
yes